packages/cactus-test-plugin-ledger-connector-besu/src/test/typescript/integration/plugin-validator-besu/sign-transaction-endpoint.test.ts

```diff
@@ -1,4 +1,5 @@
-import test, { Test } from "tape-promise/tape";
+// import test, { Test } from "tape-promise/tape";
```

Comment:
If this isn't required, please remove this

Reply:
thank you so much!